README.md

```diff
@@ -210,15 +210,15 @@ There is file `$XDG_CONFIG_HOME/pop-shell/config.json` where you can add the fol
   title: "<Optional Window Title>"
 }
 ```
-For example, doing `xprop` on GNOME Settings (or GNOME Control Center), the WM_CLASS values are `gnome-control-center` and `Gnome-control-center`. Use the second value (Gnome-control-center), which pop-shell will read. `title` is optional.
+For example, doing `xprop` on GNOME Settings (or GNOME Control Center), the WM_CLASS values are `gnome-control-center` and `Gnome-control-center`. Use the second value (Gnome-control-center), which pop-shell will read. The `title` is optional.
```

Comment:
Like `auto-tile mode` earlier, `title` is the name of a specific field here, so `The` is not necessary. Either say ``The `title` field is optional.`` so `The` can operate on `field`, or don't make this change.

Reply:
Alright! 😄